### ISSUE TITLE
Allow highlight spans base on tag in trace detail view

### ISSUE THREAD

**Issue by quaff** [jaegertracing/jaeger-ui#651]
## Requirement - what kind of business use case are you trying to solve?
I have a big trace consists of many spans, I want find every span which contains specified tag.

## Problem - what in Jaeger blocks you from solving the requirement?
Currently, I need click every span and check its tags.

## Proposal - what do you suggest to solve the problem or improve the existing situation?
Provide an input box `Highlight...` like existed `Find...` let user input tag (e.g. `component=jdbc`), then highlight all matched span by wrapping colored border.

**Comment by yurishkuro** [comment]
this is pretty much how Find already works, except that the syntax is just the search pattern, it does not recognize tag=value

**Comment by quaff** [comment]
> this is pretty much how Find already works, except that the syntax is just the search pattern, it does not recognize tag=value

Yes, It should reuse `Find...`, one more thing,  click previous/next should add highlight border for current span.

**Comment by jpkrohling** [comment]
I'm not exactly sure this is a good first issue, but I'm adding this label anyway, as apparently, there's a reference on how this can be implemented already.

**Comment by grvv** [comment]
Can I work on this one?

**Comment by yurishkuro** [comment]
@grvv give it a shot. If you find yourself going too deep with the implementation, I suggest first posting findings / screenshots here. Perhaps even start with a concrete spec of how the new functionality would look from the user point of view.

**Comment by Aaishpra** [comment]
Hey, @yurishkuro I am an outreachy applicant can I try this up?

**Comment by yurishkuro** [comment]
@Aaishpra sure. Only thing is I don't really have a good sense of the complexity here, so `good first issue` label might be a bit optimistic.

**Comment by 7908837174** [comment]
Hi @yurishkuro, I'd love to take on this issue! Please assign it to me, and I'll start working on a proposal. Thanks!

**Comment by Bhavishya07-Tech** [comment]
Hi, I'd like to work on this issue. I plan to add an input field where users can filter spans by tag and highlight matching ones with a colored border. Let me know if I can proceed.
